### PR TITLE
Enable redis password auth

### PR DESCRIPTION
**Pull request by krtk6160**

### CODE REVIEW

File: charts/galoy/values.yaml [hunk 1]
```diff
@@ -87,7 +87,7 @@ redis:
     persistence:
       enabled: true
   auth:
-    enabled: false
+    password: password
```
Comment (nicolasburtey) [inline charts/galoy/values.yaml:90]: password is automatically generated?

**Comment by krtk6160** [comment]
Nope. The redis chart differs from the mondo chart in this aspect. We need
to provide the password explicitly via the values.

File: charts/galoy/values.yaml (hunk 1 above)
Comment (krtk6160) [reply charts/galoy/values.yaml:90]: Nope. The redis chart differs from the mondo chart in this aspect. We need
to provide the password explicitly via the values.

Comment (nicolasburtey) [reply charts/galoy/values.yaml:90]: how is the password defined in that case currently? You'll manually set it?

**Review by nicolasburtey — APPROVED** [review]

File: charts/galoy/values.yaml (hunk 1 above)
Comment (krtk6160) [reply charts/galoy/values.yaml:90]: Yes, via the values.yaml. For testnet and mainnet we have our regular flow of injecting from the `configs` repo and for regtest we have it in the default values.yaml file

**Pull request merged**